.travis.yml

```diff
@@ -5,13 +5,15 @@ dist: xenial
 jobs:
   fast_finish: true
   include:
-    - sudo: required
+    - os: linux
+      dist: xenial
       env: USING_AUTOTOOLS=yes
-      before_install:
-        - sudo add-apt-repository ppa:ondrej/autotools -y # automake 1.15
-        - sudo add-apt-repository ppa:cz.nic-labs/knot-dns -y # pkg-config 0.29.2
-        - sudo apt-get update
-        - sudo apt-get install -y autoconf automake-1.15 pkg-config m4
+      addons:
+        apt:
+          packages: [libsndfile-dev, libfftw3-dev, libasound2-dev, autoconf, automake-1.15, pkg-config, m4]
+        sources:
+          - sourceline 'ppa:ondrej/autotools' # automake 1.15
+          - sourceline 'ppa:cz.nic-labs/knot-dns' # pkg-config 0.29.2
```

Comment:
you changed this in #102 already - should not be part of this PR

Reply:
This is true, but this PR predates #102 and this change was required to make the CI less flaky as it spuriously failed due to connection issues, which the apt-addon of travis workarounds by re-trying. I'll rebase and remove this once another PR is merged

Reply:
As that change was merged I removed the commit from this PR now